Minio.Examples/Program.cs

```diff
@@ -77,22 +88,18 @@ public static void Main(string[] args)
                 accessKey = "Q3AM3UQ867SPQQA43P2F";
                 secretKey = "zuf+tfteSlswRu7BJ86wekitnifILbZam1KYY3TG";
                 enableHTTPS = true;
+                port = 443;
```

Comment:
why do we need `port` for standard ports like 443 and 80?

Reply:
I made the change for more brevity in code. [Suggestion](https://github.com/minio/minio-dotnet/pull/446#discussion_r466103780).